src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -31,6 +33,13 @@ type WorkspaceTagsSettingsPageOnyxProps = {
 };
 type WorkspaceTagsSettingsPageProps = WorkspaceTagsSettingsPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAGS_SETTINGS>;
 
+function billableExpensesPending(policy: OnyxEntry<OnyxTypes.Policy>) {
+    if (policy?.disabledFields?.defaultBillable) {
+        return policy?.pendingFields?.disabledFields ?? policy?.pendingFields?.defaultBillable;
+    }
+    return policy?.pendingFields?.disabledFields;
```

Comment:
Just added, please take a peek if it's clear

Reply:
I was looking more for something like "why" it works this way. I am not sure if I understood why we have a `billableExpensesPending()` method instead of checking for our possible "pending" situations.

To give some explanation of the feature itself:

if we ever have `policy.disabledFields.defaultBillable` that means the feature is disabled entirely. If we have `policy.defaultBillable === true` then `policy.disabledFields.defaultBillable` must be `false` and that means we will always "default to billable". If `policy.defaultBillable === false` then we will default to "non-billable".

After explaining that, does it make sense to have a fallback to `policy?.pendingFields?.defaultBillable` when we are "disabled"? I would think not because the only value that matters is the `policy.pendingFields.disabledFields.defaultBillable` for that particular case.

Reply:
So I think the logic would be more like:

```js
// The field is disabled - only show the "pending" for the disabled pending action because the defaultBillable state is not relevant in this case.
if (policy.disabledFields.defaultBillable) {
    return policy.pendingFields.disabledFields;
}

// Otherwise, we are changing the `defaultBillable` so look at this field only.
return policy.pendingFields.defaultBillable;
```
WDYT?